lib/rules/disallow-multiple-var-decl.js

```diff
@@ -40,6 +40,17 @@
  *
  * for (var i = 0, j = arr.length; i < j; i++) {}
```

Comment:
Ouch, sorry. One more nit: Need to update README.

https://github.com/Receiptful/node-jscs/blob/feature/except-require/lib/rules/disallow-multiple-var-decl.js#L4-L10